packages/legend-graph/src/graph-manager/protocol/pure/v1/transformation/pureGraph/from/V1_ServiceTransformer.ts

```diff
@@ -183,6 +193,24 @@ const transformSingleExecution = (
   return execution;
 };
 
+const transformDeploymentOwnership = (
+  element: DeploymentOwnership,
+  context: V1_GraphTransformerContext,
+): V1_DeploymentOwnership => {
+  const ownership = new V1_DeploymentOwnership();
+  ownership.identifier = element.identifier;
+  return ownership;
+};
+
```

Comment:
is it possible maybe we could move these down a little bit to be closer to transformServiceOwnership()